test/index.js

```diff
@@ -101,24 +116,29 @@ describe('Schwifty', () => {
             ]
         });
 
-        getServer(config, (err, server) => {
+        getServer(config).then(function (server) {
```

Comment:
It's quite rare in async/await land that you'll actually see a `.then()` or `.catch()`.  Instead you can simply await the promise,
```js
const server = await getServer(config);
```

Reply:
oh boi... 🎺  this just blew my brain open a bit, plus looking at the haute example.

I _think_ async / await just took another click for me....I'll take another shot at this, then, I think I get it a bit more now. thanks, dude